src/helpers/__tests__/overrides.test.js

```diff
@@ -138,20 +165,46 @@ describe('Helpers - Overrides', () => {
     });
   });
 
-  test('mergeStyleOverrides', () => {
-    const overrideObject1 = {color: 'red', textTransform: 'uppercase'};
-    const overrideObject2 = {color: 'blue'};
+  test('mergeOverride can compose props functions', () => {
+    const override1 = {
+      props: () => ({prop1: 'val1', prop2: 'val2'}),
+    };
+    const override2 = {props: () => ({prop1: 'newValue'})};
+    const result = mergeOverride(override1, override2);
+    expect(typeof result.props).toBe('function');
+    // $FlowFixMe style should be a function here
```

Comment:
what does this mean?

Reply:
Sorry, I forgot to change the comment. Since `mergeOverrideds` now can compose props which are functions I added test case for that. Similar to `mergeOverride can compose style functions`
And since props can be undefined flow warn us in this particular scenario when we call props.